docs/reference/components.cue

```diff
@@ -446,6 +446,7 @@ components: {
 		telemetry: metrics: {
 			// Default metrics for each component
 			vector_events_processed_total: _vector_events_processed_total
+			vector_processed_bytes_total:  _vector_processed_bytes_total
```

Comment:
Are these metrics actually named with the `vector_` prefix or is that their official namespace? Asking because we're not including the namespace in other parts of the docs. Ex:

https://master.vector.dev/docs/reference/sources/host_metrics/#cpu_seconds_total

Reply:
As far as I know they are all prefixed with `vector_` now. I'll double check, though.

Reply:
While they are prefixed with `namespace` field, that's only done in some metric sinks, and `namespace` can be overwritten or even completely removed so I think it would be better to use their real names here and handle the whole prefixing/not prefixing `namespace` in the documentation of the individual sinks.